From: SeongJae Park <sj@kernel.org>
Date: Wed, 17 Jul 2024 16:41:58 -0700
Subject: [PATCH] ==== DAMON tests fixup ====

Misc fixups for DAMON {self,kunit} tests

This patchset is for minor fixups of DAMON selftests and kunit tests.
First three patches make DAMON selftests more cleanly maintained
(patches 1 and 2) without unnecessary warnings (patch 3).  Following six
patches remove unnecessary test case (patch 4), handle configs
combinations that can make tests fail (patches 5-7), reorganize the test
files following the new guideline (patch 8), and add reference
kunitconfig for DAMON kunit tests (patch 9).

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/eRT2FY1h | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/eRT2FY1h

diff --git a/damon_meta_changes/eRT2FY1h b/damon_meta_changes/eRT2FY1h
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

